client/src/views/EducationTasks.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div>
+    <tabs title="Onderwijstaken"></tabs>
```

Comment:
waarom heb je duplicate tabs gebruik?

Reply:
is niet dubbel is een component zo gebruik je components nou eenmaal